packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -294,21 +296,21 @@ describe('Sayt Component', () => {
       customEvent = stub(window, 'CustomEvent').returns(eventObj);
     });
 
-    it('should dispatch an event with a payload that includes the query and searchbox', () => {
-      sayt.dispatchRequestEvent(eventName, query, searchbox);
+    it('should dispatch an event with a payload that includes the query and group', () => {
+      const s = sayt.dispatchRequestEvent(eventName, query, group);
```

Comment:
Why is a constant needed here? It doesn't appear to be used below.

Reply:
Good catch. This was left over from a debug session.